eng/publish.proj

```diff
@@ -104,4 +104,56 @@
 
     <Message Importance="High" Text="Publishing %(SymbolPackagesToPublish.Identity) to $(SymbolServerPath)"/>
   </Target>
-</Project>
\ No newline at end of file
+
+  <Target Name="PublishTestAssets">
+    <Error Text="The ExpectedFeedUrl property wasn't supplied." Condition="'$(ExpectedFeedUrl)' == ''" />
+    <Error Text="The AccountKey property wasn't supplied." Condition="'$(AccountKey)' == ''" />
+    <Error Text="FilesToPublishPattern property wasn't supplied." Condition="'$(FilesToPublishPattern)' == ''" />
+
+    <PropertyGroup>
+      <_TestAssetVersion>$(PackageVersion)-$(VersionSuffix)</_TestAssetVersion>
+      <_FileRelativePathBase>corefx-tests/$(_TestAssetVersion)/$(OSGroup).$(ArchGroup)/$(TargetGroup)/</_FileRelativePathBase>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_ItemsToPush Remove="@(_ItemsToPush)" />
+      <_ItemsToPush Include="$(FilesToPublishPattern)" />
+      <_ItemsToPush>
+        <RelativeBlobPath>$(_FileRelativePathBase)$([System.String]::Copy('%(RecursiveDir)%(Filename)%(Extension)').Replace('\' ,'/'))</RelativeBlobPath>
```

Comment:
Is this trying to do what MSBuild Normalize path do?

https://github.com/dotnet/arcade/blob/beadc52f70da2965fbab941e9b4daa5ddf3e3291/src/Microsoft.DotNet.CoreFxTesting/build/core/Core.targets#L54

```suggestion
        <RelativeBlobPath>$([MSBuild]::NormalizePath('$(_FileRelativePathBase)', '%(RecursiveDir)', '%(Filename)%(Extension)'))</RelativeBlobPath>
```

Reply:
I stole that line from src/uploadtests.proj. I'll replace it with your suggestion.